src/lib/features/playground/clean-context.test.ts

```diff
@@ -29,7 +29,25 @@ test("doesn't add non-existing properties", async () => {
         appName: 'test',
     };
 
-    const output = cleanContext(input);
+    const { context: output } = cleanContext(input);
 
     expect(output).toStrictEqual(input);
 });
+
+test('it returns the names of all the properties it removed', async () => {
+    const { removedProperties } = cleanContext({
+        appName: 'test',
+        ...invalidJsonTypes,
+    });
+
+    const invalidProperties = Object.keys(invalidJsonTypes);
+
+    // verify that the two lists contain all the same elements
+    expect(removedProperties).toEqual(
+        expect.arrayContaining(invalidProperties),
```

Comment:
is it because the order is not predictable? for predictable order I'd do simple equal

Reply:
I haven't checked the order, to be honest. I could, but it's not something that I think we can intrinsically rely on; we might want to sort it alphabetically or something later 🤷🏼 I don't think that should cause the tests to fail. So this just seemed like a more robust way to check it.